src/configuration-requestor.ts

```diff
@@ -73,23 +67,37 @@ export default class ConfigurationRequestor {
     }
   }
 
+  // This is hopefully temporary workaround until we address
+  // non-unique model version of bandits for cold start
```

Comment:
im concerned about this:
1. why this is temporary?
2. what does the "proper" fix look like? 
3. why we're not doing it on the first place?

Reply:
Started a thread on slack about how we want to approach this. Proper way would be to change how modelVersion for cold start is created i.e adding banditkey suffix/prefix to modelversion for cold start. 2 reasons why this apprach is better in my opinion.
1. Less coupling of logic and data.
2. This will only require changes in API instead of handling it in each sdk.

I may be getting ahead of myself with this change, but reverting this change later on will take little to no time and the upside is that it will just work once it's merged.

Reply:
[Here](https://github.com/Eppo-exp/eppo/blob/ec80612ce41f1ba870d1aeba375242cda88a02f2/packages/api/src/bandit/services/bandit.service.ts#L112) and [here](https://github.com/Eppo-exp/eppo/blob/ec80612ce41f1ba870d1aeba375242cda88a02f2/packages/api/src/randomized-assignment/services/universal-flag-config.service.ts#L283) are the only places where I found model version assigment.  ActiveModelVersion ID is database pk from what I remember, so they should be unique?

Reply:
> between different models
Okay I just noticed this detail. If that's the case then you might be right.